camelot/view/crud_action.py

```diff
@@ -85,20 +86,23 @@ def add_data(self, model_context, row, columns, obj, data):
             items = []
             locale = model_context.locale
             for column in changed_columns:
-                # copy to make sure the original dict can be reused in
-                # subsequent calls
-                field_attributes = dict(static_field_attributes[column])
-                # the dynamic attributes might update the static attributes,
-                # if get_dynamic_field_attributes is overwritten, like in 
-                # the case of the EntityAdmin setting the onetomany fields
-                # to not editable for objects that are not persistent
-                field_attributes.update(dynamic_field_attributes[column])
-                delegate = field_attributes['delegate']
-                field_action_model_context = self.field_action_model_context(
-                    model_context, obj, field_attributes
-                )
-                item = delegate.get_standard_item(locale, field_action_model_context)
-                items.append((column, item))
+                if action_state is not None:
```

Comment:
die 'if' op action_state is hier eigenlijk nt helemaal juist, omdat in geval 'admin.list_action' == None, de action_state ook None gaat zijn

Reply:
Vervang ik dit dan door de volledige condition van daarboven?

`        if (admin.is_readable( obj ) and (data==True) and (obj is not None)):`